core/app/models/workarea/checkout.rb

```diff
@@ -161,7 +161,9 @@ def continue_as(user)
     # @return [self]
     #
     def update(params = {})
```

Comment:
We discussed a `return true if params.blank?` to mimic calling `update` on a model where nothing has changed.